### PR TITLE
Add http rbs

### PR DESCRIPTION
**Pull request by sue445**
I add rbs for https://github.com/httprb/http

### CODE REVIEW

File: gems/http/5.1/http.rbs [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Write the type definition here!
+
+module HTTP
+  interface _Stringable
+    def to_s: () -> String
+  end
```
Comment (ybiquitous) [inline gems/http/5.1/http.rbs:6]: [suggest] I'm unsure but the built-in `_ToS` interface may be available instead:

https://github.com/ruby/rbs/blob/v3.1.1/core/builtin.rbs#L13-L15

File: gems/http/5.1/_test/test.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Write Ruby code to test the RBS.
+# It is type checked by `steep check` command.
+
```
Comment (ybiquitous) [inline gems/http/5.1/_test/test.rb:3]: [suggest] It's possible to remove the scaffold comment.

```suggestion
```

File: gems/http/5.1/http.rbs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Write the type definition here!
+
```
Comment (ybiquitous) [inline gems/http/5.1/http.rbs:2]: [suggest] It's possible to remove the scaffold comment, too.

```suggestion
```

File: gems/http/5.1/http.rbs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  class Response
+    attr_reader status: Integer
+    attr_reader version: String
+    attr_reader body: String
+    attr_reader request: HTTP::Request
+    attr_reader proxy_headers: Hash[untyped, untyped]
+
+    def initialize: (Hash[Symbol, untyped] opts) -> void
+
+    # defined by def_delegator
```
Comment (ybiquitous) [inline gems/http/5.1/http.rbs:39]: [suggest] This internal comment is shown as a public doc on IDE, so it seems better to remove it.

```suggestion
```

**Review by ybiquitous — COMMENTED** [review]
@sue445 Thanks for the Pull Request! I've commented on a few points of a bit concern.

File: gems/http/5.1/http.rbs (hunk 1 above)
Comment (sue445) [reply gems/http/5.1/http.rbs]: @ybiquitous Thank you! I fixed at 3b6d7cb

**Review by ybiquitous — COMMENTED** [review]
LGTM! But I recommend getting other maintainers' approval.

**Comment by sue445** [comment]
@pocke Could you review this patch?

**Review by pocke — APPROVED** [review]

**Pull request merged**